src/passes.jl

```diff
@@ -50,12 +34,38 @@ function branches_to(b)
   return result
 end
 
+# After removing branch, none are point to the branch, but decrement pointing
+# above.
```

Comment:
This is a confusing comment.

Reply:
Probably made sense to me at the time... I'll give it another go.